conversational/ISSUE_TEMPLATE.md

```diff
@@ -28,11 +28,10 @@ https://plnkr.co or similar (you can use this template as a starting point: http
 
 
 * **Please tell us about your environment:**
-
-- version: 2.0.0-beta.X
-- Browser: [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
-- Language: [all | TypeScript X.X | ES6/7 | ES5 | Dart]
-
+  
```

Comment:
Thank you. Why do you change this paragraph?

Reply:
Because old version have no indentation for sub-items